test/dlc_tests/eks/pytorch/inference/test_eks_pytorch_inference.py

```diff
@@ -49,13 +47,19 @@ def test_eks_pytorch_neuron_inference(pytorch_inference_neuron):
             eks_utils.eks_forward_port_between_host_and_container(selector_name, port_to_forward, "8080")
 
         assert test_utils.request_pytorch_inference_densenet(
-            port=port_to_forward, server_type=server_type, model_name="pytorch-resnet-neuron"
+            port=port_to_forward, server_type=server_type, model_name=model_name
         )
     finally:
         run(f"kubectl delete deployment {selector_name}")
         run(f"kubectl delete service {selector_name}")
 
 
+@pytest.mark.model("resnet")
```

Comment:
is this supposed to be neuron or neuronx? If neuron, lets create a separate PR for this as this PR concerns neuronx only

Reply:
I had to restructure the code here to allow for neuronx without copying any code. The initial test_eks_pytorch_neuron_inference function below has been broken into: a common part  __run_pytorch_neuron_inference and two tests (the neuronx one has just been added in the latest commit).